{"name": "Проверка создания пользователя без заполнения поля пароля", "status": "passed", "start": 1712519895665, "stop": 1712519896788, "uuid": "03356a58-04ca-41b5-b879-c03350b1cce3", "historyId": "a629b2405f7d77772026d053b349c129", "testCaseId": "a629b2405f7d77772026d053b349c129", "fullName": "tests.test_create_new_user.TestCreateNewUser#test_create_new_user_without_password_data_error_403", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_create_new_user"}, {"name": "subSuite", "value": "TestCreateNewUser"}, {"name": "host", "value": "WIN-R1RFFME11IE"}, {"name": "thread", "value": "7708-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_create_new_user"}]}